src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetDateCommand.cs

```diff
@@ -393,7 +393,7 @@ private string UFormatDateString(DateTime dateTime)
                             break;
 
                         case 'l':
-                            sb.Append(StringUtil.Format("{0,2:0}", dateTime.Hour%12));
+                            sb.Append("{0:%h}");
```

Comment:
Using http://man7.org/linux/man-pages/man1/date.1.html as reference, this should be space padded: `{0,2:%h}`

Reply:
Yes.
Fixed.